### PR TITLE
feat : video-listing-page

### PR DESCRIPTION
**Pull request by mikesch803**
1. video-listing-page is added
2.  routing btw video-listing-page and home-page is added (as of now navigate only navbar)
3.  getting the data from the backend while loading for the first time in video-listing-page api is added

# files to check

- [ ] src/App.js
- [ ] src/components/header/Header.jsx
- [ ] src/index.js
- [ ] src/pages/video-listing-page/VideoListing.jsx

# files not to check
a lot of files are empty please ignore that and other files

## [live link](https://extraordinary-tulumba-1a066e.netlify.app) **CLICK ON TREND**

### CODE REVIEW

File: src/pages/video-listing-page/VideoListing.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }, []);
+
+  return (
+    <div className="trend-video-grid-layout">
+      <Aside />
+      <main className="trend-main">
+        <h2 className="trend-title">Trending Videos</h2>
+        <div className="video-container">
+          {videos.map((video) => (
+            <div className="card card-vrt" key={video._id}>
```
Comment (Vishnu-Aithal) [inline src/pages/video-listing-page/VideoListing.jsx:25]: suggestion - you can extract video card as a separate component

Comment (mikesch803) [reply src/pages/video-listing-page/VideoListing.jsx:25]: will separate out soon as files changes

File: src/pages/video-listing-page/VideoListing.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        min-height: 100vh;
+        grid-template-areas:
+          "header header header"
+          "aside main main";
+      }
+
+.trend-main {
+    grid-area: main;
+    /* display: flex;
+    flex-direction: column; */
```
Comment (Dev-AkshatGupta) [inline src/pages/video-listing-page/VideoListing.css:13]: Try to remove the commented code

**Review by Dev-AkshatGupta — APPROVED** [review]
Seems cool to me doesn't think there's much to comment about

**Pull request merged**